csharp/ql/lib/semmle/code/csharp/security/dataflow/flowsinks/Html.qll

```diff
@@ -56,13 +56,11 @@ class HtmlTextWriterSink extends HtmlSink {
 }
 
 /**
- * An expression that is used as an argument to an HTML sink method on
- * `AttributeCollection`.
+ * DEPRECATED: Attribute collections are no longer considered HTML sinks.
  */
-class AttributeCollectionSink extends HtmlSink {
+deprecated class AttributeCollectionSink extends DataFlow::ExprNode {
   AttributeCollectionSink() {
     exists(SystemWebUIAttributeCollectionClass ac, Parameter p |
-      p = ac.getAddMethod().getParameter(1) or
```

Comment:
Why is this removed?

Reply:
Bah, stupid mistake (first just removing it for testing and then deprecated the stuff afterwards). Thank you for catching.